blackbox/docs/general/dql/joins.rst

```diff
@@ -201,17 +201,48 @@ Example with ``within`` scalar function::
     +--------------+---------+
     SELECT 2 rows in set (... sec)
 
-However, there are no specific optimisations for certain join conditions such
-as ``=`` (equals) yet. The JOIN operation is implemented as a generic nested
-loop that invokes the operation on every record of the left table with every
-record of the right table. However, these filters are applied in a distributed
-way for each shard of the used tables.
-
 Limitations
 ...........
```

Comment:
thinks this should be kept as last paragraph. but please fix header level here as well